src/nvim/option.c

```diff
@@ -4620,9 +4603,9 @@ void *get_option_varp_scope_from(OptIndex opt_idx, int scope, buf_T *buf, win_T
 
 void *get_varp_from(vimoption_T *p, buf_T *buf, win_T *win)
 {
-  // hidden option, always return NULL
-  if (p->var == NULL) {
-    return NULL;
+  // hidden options always use the same var pointer
```

Comment:
should we add an `p->var != NULL` here ?

Reply:
> should we add an `p->var != NULL` here ? and the other functions where `p->var == NULL` was being checked.

`p->var` cannot be NULL anymore, unless something has gone seriously wrong